test/issues/0468.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+
+var assert = require('assert');
+var yaml = require('../..');
+
+test('should not indent arrays an extra level when disabled', function () {
+  var output = yaml.dump([{"a":"a_value","b":"b_value"},{"a":"a2_value","expanded":true,"items":[{"a":"a_a_value","b":"a_b_value"}]}], { noArrayIndent: true });
```

Comment:
It's better to write such objects in nested form, starting from newline:

```js
var output = yaml.dump(
  [
    {
      a: "a_value",
....
  ],
```

Also probably worth shorten `a_value` => `a_val` to minimize useless noise.

Reply:
I will fix.